src/ring_theory/noetherian.lean

```diff
@@ -663,6 +663,11 @@ begin
   refine (submodule.restrict_scalars_embedding R S M).dual.well_founded h
 end
 
+instance ideal.quotient.is_noetherian_ring {R : Type*} [comm_ring R] [h : is_noetherian_ring R]
+  (p : ideal R) : is_noetherian_ring p.quotient :=
+is_noetherian_ring_iff.mpr $ is_noetherian_of_tower R $
+  is_noetherian_of_quotient_of_noetherian _ _ _ (is_noetherian_ring_iff.mp h)
+
```

Comment:
Is there a similar statement for when `p : submodule R M` and `is_noetherian R M`? If so, I think this proof falls out as a special case of it.